drivers/video/ov5640.c

```diff
@@ -359,14 +362,117 @@ static const struct ov5640_reg init_params[] = {
 	{0x5000, 0xa7},
 };
 
-static const struct ov5640_reg low_res_params[] = {
+static const struct ov5640_reg init_params_dvp[] = {
+	{0x4740, 0x21},
+	{0x4050, 0x6e},
+	{0x4051, 0x8f},
+	{0x3017, 0xff},
+	{0x3018, 0xff},
+	{0x302c, 0x02},
+	{0x3108, 0x01},
+	{0x3630, 0x2e},
+	{0x3a18, 0x00},
+	{0x3a19, 0xf8},
+	{0x3635, 0x1c},
+	{0x3c04, 0x28},
+	{0x3c05, 0x98},
+	{0x3c06, 0x00},
+	{0x3c07, 0x08},
+	{0x3c08, 0x00},
+	{0x3c09, 0x1c},
+	{0x3c0a, 0x9c},
+	{0x3c0b, 0x40},
+	{TIMING_TC_REG20_REG, 0x47},
+	{TIMING_TC_REG21_REG, 0x01},
+	{0x3800, 0x00},
```

Comment:
If I understand correctly, the way to initialize DVP is to first initialize MIPI/CSi2, then change registers to become DVP.

This is not a problem though. If someone bother the small extra ROM over head or initialization time they can then send a patch :)